ONSdigital/babbage
# Created intro meta partial for displaying last updated

### What

Created an intro-meta partial that displays the last updated value, if present, underneath the title for static pages.

### How to review

- Visit an ONS static page, such as the Accessibility page
- Check that the last updated partial renders and matches the prototype as per the [ticket](https://trello.com/c/iMEB9YSh/4665-add-last-updated-to-static-help-pages-1-2)

- If there is no "Last updated" section, you may need to add a date to the page in Florence

### Who can review

Anyone but me
